src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  padding: 0;
+  text-transform: uppercase;
+  display: flex;
+  height: 60px;
+  padding-left: 20px;
+  justify-content: space-between;
+  padding-right: 50px;
+  &__logo {
+    width: 40px;
+    height: 40px;
+    margin: auto 30px;
+  }
+}

```

Comment:
```suggestion
.header {
  padding: 0;
  text-transform: uppercase;
  display: flex;
  height: 60px;
  padding-left: 20px;
  justify-content: space-between;
  padding-right: 50px;
  
  &__logo {
    width: 40px;
    height: 40px;
    margin: auto 30px;
  }
}

```

Reply:
add align-items to header, so you won't need to center the logo with margin auto